javascript/apps/taiga/src/app/shared/comments/components/comment-detail/comment-detail.component.html

```diff
@@ -0,0 +1,99 @@
+<!--
+This Source Code Form is subject to the terms of the Mozilla Public
+License, v. 2.0. If a copy of the MPL was not distributed with this
+file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+Copyright (c) 2023-present Kaleidos INC
+-->
+
+<ng-container *ngIf="model$ | async as vm">
+  <ng-container
+    *ngIf="
+      vm.project.userIsAdmin ||
+      (user.username === comment.createdBy.username && canComment)
+    ">
+    <ng-container *transloco="let t">
+      <tui-hosted-dropdown
+        [tuiDropdownAlign]="'right'"
+        [content]="showDeleteCommentConfirm ? confirmDelete : commentOptions"
+        [(open)]="commentOptionsState">
+        <button
+          role="menu"
+          appearance="action-button-2"
```

Comment:
This button should have border-radius 2px but it doesn't have it, as it's a component, confirm first with the UI team